target/linux/x86/base-files/etc/board.d/02_network

```diff
@@ -45,34 +45,84 @@ roqos-roqos-core-rc10)
+	ucidef_set_network_device_path "eth5" "pci0000:00/0000:00:01.0/0000:01:00.0/0000:02:03.0/0000:04:00.0"
+	ucidef_set_network_device_path "eth6" "pci0000:00/0000:00:02.0/0000:05:00.0/0000:06:02.0/0000:07:00.0"
+	ucidef_set_network_device_path "eth7" "pci0000:00/0000:00:02.0/0000:05:00.0/0000:06:03.0/0000:08:00.0"
+	ucidef_set_interfaces_lan_wan  "eth0 eth2 eth3 eth4 eth5 eth6 eth7" "wan"
+	;;
+sophos-xg-125r2|sophos-xg-125wr2| \
+sophos-xg-135r2|sophos-xg-135wr2)
+	ucidef_set_network_device_path "eth1" "pci0000:00/0000:00:14.0"
+	ucidef_set_network_device_path "wan"  "pci0000:00/0000:00:14.1"
+	ucidef_set_network_device_path "eth3" "pci0000:00/0000:00:14.2"
+	ucidef_set_network_device_path "eth4" "pci0000:00/0000:00:14.3"
+	ucidef_set_network_device_path "eth5" "pci0000:00/0000:00:01.0/0000:01:00.0/0000:02:02.0/0000:03:00.0"
+	ucidef_set_network_device_path "eth6" "pci0000:00/0000:00:01.0/0000:01:00.0/0000:02:03.0/0000:04:00.0"
+	ucidef_set_network_device_path "eth7" "pci0000:00/0000:00:02.0/0000:05:00.0/0000:06:02.0/0000:07:00.0"
+	ucidef_set_network_device_path "eth8" "pci0000:00/0000:00:02.0/0000:05:00.0/0000:06:03.0/0000:08:00.0"
+	ucidef_set_interfaces_lan_wan  "eth1 eth3 eth4 eth5 eth6 eth7 eth8" "wan"
+	;;
+sophos-sg-135r3|sophos-sg-135wr3)
+	ucidef_set_network_device_path "eth0" "pci0000:00/0000:00:17.0/0000:0c:00.0"
+	ucidef_set_network_device_path "wan"  "pci0000:00/0000:00:16.0/0000:0b:00.1"
+	ucidef_set_network_device_path "eth2" "pci0000:00/0000:00:16.0/0000:0b:00.0"
+	ucidef_set_network_device_path "eth3" "pci0000:00/0000:00:17.0/0000:0c:00.1"
+	ucidef_set_network_device_path "eth4" "pci0000:00/0000:00:09.0/0000:02:00.0"
+	ucidef_set_network_device_path "eth5" "pci0000:00/0000:00:0a.0/0000:03:00.0"
+	ucidef_set_network_device_path "eth6" "pci0000:00/0000:00:0b.0/0000:04:00.0"
+	ucidef_set_network_device_path "eth7" "pci0000:00/0000:00:0c.0/0000:05:00.0"
+	ucidef_set_interfaces_lan_wan  "eth0 eth2 eth3 eth4 eth5 eth6 eth7" "wan"
+	;;
+sophos-xg-135r3|sophos-xg-135wr3)
+	ucidef_set_network_device_path "eth1" "pci0000:00/0000:00:17.0/0000:0c:00.0"
+	ucidef_set_network_device_path "wan"  "pci0000:00/0000:00:16.0/0000:0b:00.1"
+	ucidef_set_network_device_path "eth3" "pci0000:00/0000:00:16.0/0000:0b:00.0"
+	ucidef_set_network_device_path "eth4" "pci0000:00/0000:00:17.0/0000:0c:00.1"
+	ucidef_set_network_device_path "eth5" "pci0000:00/0000:00:09.0/0000:02:00.0"
+	ucidef_set_network_device_path "eth6" "pci0000:00/0000:00:0a.0/0000:03:00.0"
+	ucidef_set_network_device_path "eth7" "pci0000:00/0000:00:0b.0/0000:04:00.0"
+	ucidef_set_network_device_path "eth8" "pci0000:00/0000:00:0c.0/0000:05:00.0"
+	ucidef_set_interfaces_lan_wan  "eth1 eth3 eth4 eth5 eth6 eth7 eth8" "wan"
 	;;
 traverse-technologies-geos)
 	ucidef_set_interface_lan "eth0 eth1"
 	ucidef_add_atm_bridge "0" "35" "llc" "bridged"
 	ucidef_set_interface_wan "nas0" "dhcp"
-	macaddr="$(cat /sys/class/net/eth0/address)" 2>/dev/null
+	macaddr="$(cat /sys/class/address)" 2>/dev/null
```

Comment:
This is proper?

Reply:
No, this is my mistake not picking latest changes, thanks for bringing it up, I'll force-push the fix.